### PR TITLE
structurelib interation and disable some multis

### PR DESCRIPTION
**Pull request by GlodBlock**

### CODE REVIEW

**Comment by draknyte1** [comment]
Why bother?

**Comment by GlodBlock** [comment]
at least gt++ is still in this pack

**Comment by GlodBlock** [comment]
i have disabled LFTR and some other unfinished mulit.
and i didn't required maintenance hatch on Algae farm/PSS

**Comment by draknyte1** [comment]
> i have disabled LFTR and some other unfinished mulit.
> and i didn't required maintenance hatch on Algae farm/PSS

LFTR works fine. 
Disabling things people may have isn’t what this PR says it does.

File: src/Java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/base/GregtechMeta_MultiBlockBase.java [hunk 1]
```diff
@@ -1727,6 +1650,9 @@ public void fixAllMaintenanceIssue() {
 		this.mScrewdriver = true;		
 	}
 
+	public boolean checkHatch() {
+		return mMaintenanceHatches.size() <= 1 && !mMufflerHatches.isEmpty();
```
Comment (Glease) [inline src/Java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/base/GregtechMeta_MultiBlockBase.java:1654]: Just curious. Is there any non polluting GT++ multi?

Comment (GlodBlock) [reply src/Java/gtPlusPlus/xmod/gregtech/api/metatileentity/implementations/base/GregtechMeta_MultiBlockBase.java:1654]: pss, chemical plant, farmer

**Comment by Dream-Master** [comment]
GlodBlock — Today at 11:51 AM
dream, i think we can merge that structurelib-interation pr in gtpp repo to test it on zeta.
i have test it on my own save and did not break anything.
DreamMasterXXL — Today at 12:23 PM
ok

**Pull request merged**